gems/pending/disk/modules/MSVSDiffDisk.rb

```diff
@@ -50,7 +50,6 @@ def d_init
       #
       # TODO: need details on Mac Alias Blob.
       #
-      # getParentPathMac(locator)
```

Comment:
@jrafanie absolutely.  I didn't think we needed it but didn't ask before pushing it.